packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts

```diff
@@ -160,7 +160,7 @@ export class SearchableModelTransformer extends TransformerPluginBase {
     }
     const fieldName = searchFieldNameOverride
       ? searchFieldNameOverride
-      : graphqlName(`search${plurality(toUpper(definition.name.value), ctx.featureFlags.getBoolean('improvePluralization', true))}`);
+      : graphqlName(`search${plurality(toUpper(definition.name.value), true)}`);
```

Comment:
what is the point of the feature flag if once true, is not compatible with the iOS library. Let's say we saw this compatibility issue with the library

- Is there a larger issue here that the Amplify iOS library depends on how the APIs that are provisioned? There are many examples of this, such as DataStore uses a query called `"sync" + pluralName`, what happens when this is changed under a feature flag, and then eventually turned to true? All new provisionings won't work with the library anymore. Can the library introspect the GraphQL API at runtime instead of hardcoding specific names?

- Do we need the same feature flag in the library that corresponds to the CLI's feature flag so that developers can upgrade both at the same time?